RP-1.netkan

```diff
@@ -1,120 +1,110 @@
-		{ "name" : "ProceduralParts" },
-		{ "name" : "RCSBuildAid" },
-		{ "name" : "RCSBuildAidCont" },
-		{ "name" : "RealAntennas" },
-		{ "any_of": [
-			{ "name" : "TestFlight" },
-			{ "name" : "TestLite" }
-		] },
-		{ "name" : "RSSVE-Textures" },
-		{ "name" : "TransferWindowPlannerFork" },
-		{ "name" : "Waterfall" },
-		{ "name" : "ModularLaunchPads" }
-	],
-	"suggests" : [
-		{ "name" : "BluedogDB" },
-		{ "name" : "FASA" },
-		{ "name" : "TextureReplacer" },
-		{ "name" : "SemiSaturatableRW" },
-		{ "name" : "JanitorsCloset" },
-		{ "name" : "HyperEdit" },
-		{ "name" : "EditorExtensionsRedux" },
-		{ "name" : "KerbalEngineerRedux" },
-		{ "name" : "kOS" },
-		{ "name" : "RealScaleBoosters" },
-		{ "name" : "NearFutureSolar" },
-		{ "name" : "NearFutureConstruction" },
-		{ "name" : "InternalRCS" },
-		{ "name" : "ProbesPlus" },
-		{ "name" : "ATKPropulsionPack" },
-		{ "name" : "WaypointManager" },
-		{ "name" : "SCANsat" },
-		{ "name" : "NavHudRenewed" },
-		{ "name" : "ConformalDecals" },
-		{ "name" : "CanaveralPads" }
-	],
-	"conflicts" : [
-		{ "name" : "RP-0" },
-		{ "name" : "KerbalConstructionTime" },
-		{ "name" : "CrowdSourcedScience" },
-		{ "name" : "HideEmptyTechNodes" },
-		{ "name" : "MechJebForAll" },
-		{ "name" : "NearFutureElectrical-DecayingRTGs" },
-		{ "name" : "KRASH" },
-		{
-			"name": "ROCapsules",
-			"max_version" : "v1.99.0.0"
-		},
-		{
-			"name": "ROEngines",
-			"max_version" : "v1.99.0.0"
-		},
-		{
-			"name": "ROSolar",
-			"max_version" : "v1.99.0.0"
-		}
-	]
-}
+spec_version: v1.26
+identifier: RP-1
+name: Realistic Progression One (RP-1)
+abstract: >-
+  Career for Realism Overhaul. Do not select/install this directly, instead
+  select the RP-1 Express Install and let it install this automatically unless
+  you know what you're doing and can fix it yourself.
+license: CC-BY-NC-SA-4.0
+author:
+  - RP-1 Group
+$kref: '#/ckan/github/KSP-RO/RP-1'
+$vref: '#/ckan/ksp-avc'
+release_status: stable
+resources:
+  manual: https://github.com/KSP-RO/RP-1/wiki
+  homepage: https://forum.kerbalspaceprogram.com/index.php?/topic/190040-*
+conflicts:
+  - name: RP-0
+  - name: KerbalConstructionTime
+  - name: CrowdSourcedScience
+  - name: HideEmptyTechNodes
+  - name: MechJebForAll
+  - name: NearFutureElectrical-DecayingRTGs
+  - name: KRASH
+  - name: ROCapsules
+    max_version: v1.99.0.0
+  - name: ROEngines
+    max_version: v1.99.0.0
+  - name: ROSolar
+    max_version: v1.99.0.0
+depends:
+  - name: ModuleManager
+  - name: RealismOverhaul
+    min_version: v16.0.0.0
+  - name: SXTContinued
+  - name: ContractConfigurator
+  - name: CustomBarnKit
+  - name: VenStockRevamp-NewParts
+  - name: RealSolarSystem
+  - name: MagiCore
+  - name: ClickThroughBlocker
+  - name: ToolbarController
+  - name: Kerbalism-Config-RO
+  - name: KSPCommunityFixes
+  - name: KerbalChangelog
+recommends:
+  - name: KSPBurst-Full
+  - any_of:
+    - name: ReStock
+    - name: VenStockRevamp
+  - any_of:
```

Comment:
I think you only need one of these--copypasta?

Reply:
The next couple lines then offer the choice between ReStock **Plus** and Ven's. I wasn't sure of the best way to express that decision. But some testing reveals that during installation there's functionally no difference between that formatting and just recommending restock plus separately: the user has to toggle all 3 mods to switch to Ven's during the install regardless. I have simplified it.